.gitattributes

```diff
@@ -0,0 +1 @@
+/ui/web/bindata.go binary
```

Comment:
this is interesting. to be clear, the effect of this is that it will show "file differs" but not actual differences right, and i see things like `git add -p` and `git checkout -p` don't apply to this file. personally I liked to be able to see changes to this file, but maybe not rationally. it makes sense to just run go generate , see the file changed, commit it, without having to actually know what the diff was.